src/wr_logic_ai/CMakeLists.txt

```diff
@@ -51,31 +52,31 @@ find_package(catkin REQUIRED COMPONENTS
 ## Generate messages in the 'msg' folder
 add_message_files(
   FILES
-  NavigationMsg.msg
   ObstaclesMsg.msg
+  NavigationState.msg
   TargetMsg.msg
-  NavigationStateMsg.msg
 )
 
 ## Generate services in the 'srv' folder
-# add_service_files(
-#   FILES
-#   Service1.srv
-#   Service2.srv
-# )
+add_service_files(
+   FILES
+   EmptySrv.srv
```

Comment:
See notes on file.

Reply:
Addressed in [277b00a](https://github.com/WisconsinRobotics/WRover_Software/pull/48/commits/277b00a47c8537aba6ba2aeeb6c09d9b5ea4d3eb).